tests/check_toml_test.py

```diff
@@ -23,6 +23,7 @@ def test_toml_bad(tmpdir):
 [owner]
 name = "John"
 dob = 1979-05-27T07:32:00-08:00 # First class dates
+favorite_letter = "Á"
```

Comment:
this test will fail in the same way as the issues is -- I'd suggest writing a separate test which does `.write_binary('...'.encode('UTF-8')`

note that `Á` is not a good choice for a UTF-8-compatible test since that's encodable in latin1 / cp1252 I usually pick snowman (`\u2603`) or upside down face (`\U0001f643`)

Reply:
Sure, will hack on this